src/Service/DocumentUploader.php

```diff
@@ -70,13 +70,22 @@ public function createDocumentFromPath(string $filePath, string $documentName, s
      */
     public function createDocumentFromUploadedFile(UploadedFile $file, string $documentName, string $documentType): Document
     {
-        $tempDir = sys_get_temp_dir();
-        $file = $file->move($tempDir, $file->getClientOriginalName());
+        try {
+            $tempDir = sys_get_temp_dir();
+            $file = $file->move($tempDir, $file->getClientOriginalName());
 
-        $document = $this->createDocumentFromPath($file->getRealPath(), $documentName, $documentType);
+            $tempPath = $file->getRealPath();
 
-        // Explicitly indicate that this is created manually, i.e. display original file name on document index page.
-        $document->setIsCreatedManually(true);
+            $document = $this->createDocumentFromPath($file->getRealPath(), $documentName, $documentType);
+
+            // Explicitly indicate that this is created manually, i.e. display original file name on document index page.
+            $document->setIsCreatedManually(true);
+        } finally {
+            $myfile = fopen('testfile.txt', 'w');
+            fwrite($myfile, $tempPath);
+            fclose($myfile);
```

Comment:
You may want to remove this …

Reply:
Yep.. whoopsie f71f3751b687960645c385b601ad00f70c259ca1